.circleci/config.yml

```diff
@@ -307,7 +307,7 @@ jobs:
             - ./dist/*.whl
             - ./dist/*.tar.gz
 
-  integration-test-integration-airflow:
+  integration-test-integration-airflow-1:
```

Comment:
It'd be great if we can use the following CI job names to easily know which version of Airflow we were running integration tests for:

```
integration-test-integration-airflow-1.x
integration-test-integration-airflow-2.x
```

Note sure of these names are valid though ;)

Reply:
`integration-test-integration-airflow-1.10` - ok.

For Airflow 2's tests, the image used is appended to the end of job name - it looks like that: 
https://app.circleci.com/pipelines/github/OpenLineage/OpenLineage?branch=airflow%2Ftasklistener&filter=all

Reply:
We can't use dots in job names:
```
**ERROR IN CONFIG FILE:[#/jobs/integration-test-integration-airflow-1.10] string [integration-test-integration-airflow-1.10] does not match pattern ^[A-Za-z][A-Za-z\s\d_-]*$**
```

Reply:
I think appending the image tag to the job name is better than simply having `-2` appended to the job name. The only think now is we get job names like:

```
integration-test-integration-airflow-gcr.io/speedy-vim-308516/apache/airflow:latest
integration-test-integration-airflow-apache/airflow:2.2.3
...
```

which are long, but also descriptive so that isn't necessarily a bad thing.